### PR TITLE
Introducing LagThreshold as a Kafka Trigger Attribute for scaling purposes.

### PR DESCRIPTION
**Pull request by aloiva**
This PR introduces a new Kafka Trigger Attribute for Kafka functions.

Changes:
- Exposed an important scaling parameter `LagThreshold` to let a user configure their scaling preferences within Kafka Trigger function.
- Updated Kafka Extension nuget package to most recent version. [ yet to be released ]

Impact:
- Improves user customisation.

### CODE REVIEW

File: extensions/Worker.Extensions.Kafka/src/KafkaTriggerAttribute.cs [hunk 1]
```diff
@@ -103,6 +103,12 @@ public KafkaTriggerAttribute(string brokerList, string topic)
         /// </summary>
         public string SslKeyPassword { get; set; }
 
+        /// <summary>
+        /// Lag threshold
+        /// Default: 1000
+        /// </summary>
+        public long LagThreshold { get => lagThreshold.GetValueOrDefault(1000L); set => lagThreshold = value; }
```
Comment (jviau) [inline extensions/Worker.Extensions.Kafka/src/KafkaTriggerAttribute.cs:110]:
```suggestion
        public long LagThreshold { get; set; } = 1000;
```

**Comment by liliankasem** [comment]
@aloiva is this PR ready to publish or is this blocked on something?

**Comment by aloiva** [comment]
Hey @liliankasem, we were waiting for Kafka Extension version 3.9 which has been released only a couple of days ago. It's ready to be published now.

**Review by liliankasem — APPROVED** [review]

**Review by liliankasem — APPROVED** [review]
lgtm; please also update release notes: https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.Kafka/release_notes.md

**Pull request merged**